datajoint/base_relation.py

```diff
@@ -260,7 +261,7 @@ def make_attribute(name, value):
                 sql = 'INSERT IGNORE'
             else:
                 sql = 'INSERT'
-            attributes = (a for a in attributes if a[0])   # omit dropped attributes
+            attributes = (a for a in attributes if a[0] is not None)  # omit dropped attributes
```

Comment:
This statement is not the problem. I just added `is not None` because it is more explicit. However, I'll add a test case for the bug above.